docs/contributing/kubernetes.md

```diff
@@ -0,0 +1,48 @@
+# Kubernetes Considerations for Tracee Contributors
+
+1. **Set up a Local Kubernetes Cluster**
+
+    We recommend using a local Kubernetes cluster for development and testing. Popular options include:
+
+    * **kind:** Kubernetes IN Docker
+    * **minikube:** Runs a single-node cluster in a VM
+    * **MicroK8s:** Lightweight, snap-based Kubernetes
+
+    Tracee's Makefile provides convenient targets for setting up MicroK8s:
+
+    ```bash
+    make -f builder/Makefile.k8s help
```

Comment:
Some considerations:
1. maybe would be good to add the output;
2. I think we should add some instructions do setup the cluster and point to a link with those instructions;
3. Please check the `manifests` and `generate`, it seems to not be working (maybe we need to open an issue)

Reply:
@rscampos, I'm having trouble building Tracee with the Kubernetes Makefile. I tried updating the Go version in the Dockerfile and running `make` from Ubuntu shell, but I'm still hitting errors. Could you offer some guidance?